CHANGELOG.md

```diff
@@ -2,7 +2,33 @@
 
 ## Next (upcoming release)
 
-TBD
+### 🔥 Performance Improvements
+* CUDA performance improvements. This results in about a 20-30% performance improvement. https://github.com/risc0/risc0/pull/2054
+* Add recursion preflight support for checked bytes/bigint https://github.com/risc0/risc0/pull/1973
+
+### 🛠 Fixes
+
+* Improve zero-knowledge property of the prover by increasing the amount of noise. We have used techniques described in this [paper](https://eprint.iacr.org/2024/1037.pdf) to strengthen the ZK property of the prover. This change results in a new set of control IDs. See the breaking changes section to learn more about this minor breaking change. https://github.com/risc0/risc0/pull/1970
```

Comment:
Maybe also link to our ZK section of our docs?
```suggestion
* Improve zero-knowledge property of the prover by increasing the amount of noise. We have used techniques described in this [paper](https://eprint.iacr.org/2024/1037.pdf) to strengthen the ZK property of the prover. See [these docs](https://dev.risczero.com/api/security-model#zero-knowledge-proving) for more information about zero knowledge and our proving system. This change results in a new set of control IDs. See the breaking changes section to learn more about this minor breaking change. https://github.com/risc0/risc0/pull/1970
```

Reply:
@tzerrell this is excellent. Thank you!